Products/CMFPlone/browser/login/password_reset.py

```diff
@@ -109,8 +110,9 @@ def _auto_login(self, userid, password):
             # with the given userid
             user = getSecurityManager().getUser()
 
-        login_time = user.getProperty('login_time', None)
-        if login_time is None:
+        default = DateTime('2000/01/01')
+        login_time = user.getProperty('login_time', default)
+        if login_time == default:
```

Comment:
If I understand your PR description correctly you say both is possible, None and default time. But here you check only default time.